src/cmd/singularity/cli/singularity.go

```diff
@@ -103,10 +104,13 @@ var SingularityCmd = &cobra.Command{
 // flags appropriately. This is called by main.main(). It only needs to happen
 // once to the root command (singularity).
 func ExecuteSingularity() {
+	defaultEnv := "/bin:/usr/bin:/usr/local/bin:/sbin:/usr/sbin:/usr/local/sbin"
+
```

Comment:
Is it not customary to have /usr/local in front of /bin and /usr/bin?

Reply:
That depends on where you come from. Some places want "system" locations to always win over /usr/local/bin if a binary of the same name exists. I'm from that school of thought.  *shrug*

Reply:
@yhcote @jmstover PATH search order fixed

Reply:
Well the point of having /usr/local in front is that you can install software without modifying what's on the system in /bin and usr/bin and make sure "your" version is found first. Not just "extra" software what wouldn't be installed on a common system. If you feel confident I'll let it go. Just wanted to point it out.